components/wmtk_components/embedded_component/internal/EmbeddedRemeshing.cpp

```diff
@@ -0,0 +1,117 @@
+#include "EmbeddedRemeshing.hpp"
+
+namespace wmtk::components::internal {
+EmbeddedRemeshing2D::EmbeddedRemeshing2D(
+    Eigen::MatrixXi& E_,
+    Eigen::MatrixXd& V_,
+    double blank_rate_,
+    double resolute_area_)
+    : E(E_)
+    , V(V_)
+    , blank_rate(blank_rate_)
+    , resolute_area(resolute_area_)
+{
+    F.resize(0, 0);
+    for (int i = 0; i < V.rows(); ++i) {
+        markedV.push_back(i);
```

Comment:
When you know the size a vector is supposed to have, you can reserve it before using `push_back`:
```
markedV.reserve(V.rows());
```
That is a bit more efficient because you avoid automatic resizing.

Besides that, we are using `emplace_back` instead of `push_back` in the toolkit. It does exactly the same but for consistency, please also use `emplace_back`.